extensions/experiences/create_iis/setup.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Labels
+#
+controller_labels:
+  - name: infra.windows_ops
+    organization: "{{ organization | default('Default') }}"
+  - name: create_iis
+    organization: "{{ organization | default('Default') }}"
+  - name: run_create_iis
+    organization: "{{ organization | default('Default') }}"
+
+# Execution Environments
+#
+controller_execution_environments:
+  # - name: apd-ee-25
+  #   description: apd-ee-25
+  #   image: quay.io/ansible-product-demos/apd-ee-25
+  #   pull: always
+  - name: apd-ee-25-rhel
```

Comment:
Why comment the "main" APD image here? The -rhel tag seems off in this context.

Reply:
Good point.  You can see I copy and pasted this from the rhel experience.